client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -114,10 +115,10 @@
 
   const AGGREGATE_STYLE = {
     node: {
-      fill: Colors.NODES.AGGREGATE,
+      fill: '#EBCB8B',
```

Comment:
Why this change?

Reply:
`Colors.NODES.AGGREGATE` has been deleted at some point so the colour code is no longer there causing a problem where the aggregate run looks the same as the current run